external/js/cothority/src/network/connection.ts

```diff
@@ -1,7 +1,8 @@
 import { Message, util } from "protobufjs/light";
-import shuffle from "shuffle-array";
+import { sprintf } from "sprintf-js";
```

Comment:
Hmmm I missed when this has been added. Please remove it from this file and from the dependencies (package + @types). There is just no point (or prove me wrong) to have that. You can simply use the \`${var}\` formatting.

Reply:
@ineiti it's marked as resolved but still there ?

Reply:
what? You want me to actually `git push`?